### PR TITLE
Bug 1516353 - import tc-terraform

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Comment by djmitche** [comment]
ok, finally green, plz 2 review

File: .taskcluster.yml [hunk 1]
```diff
@@ -53,21 +53,34 @@ tasks:
...
-              each(other):
-                name: ${other}
-                command: yarn ${other}
+            # for "packages", run tests for that package
             - $map: {$eval: packages}
               each(package):
                 name: ${package}
+                image: node:${node}
                 command: yarn workspace ${package} test
+            # for "others", do whatever they say to do
```
Comment (imbstack) [inline .taskcluster.yml]: neat

File: test/meta_test.js [hunk 2]
```diff
@@ -27,6 +30,10 @@ suite('Repo Meta Tests', function() {
     assert(extra.length === 0, `${warning} Remove: ${JSON.stringify(extra)}`);
   });
 
+  test('Node version in .taskcluster.yml matches that in package.json', function() {
```
Comment (imbstack) [inline test/meta_test.js]: :smile:

**Comment by imbstack** [comment]
For someone who is frustrated with how little work they got done in the last few days, you've been quite productive.

**Comment by imbstack** [comment]
Oh, perhaps we should make a new CODEOWNERS team for this too?

**Comment by djmitche** [comment]
I think the @taskcluster/services-reviewers team is a good choice for this repo.  I'll move tc-builder there eventually, too.

**Pull request merged**